sympy/core/tests/test_numbers.py

```diff
@@ -218,6 +218,17 @@ def test_igcd():
         raises(ValueError, lambda: igcd(*args))
 
 
+def test_igcd_lehmer():
```

Comment:
Is it possible to add more tests, to check every break condition in the algorithm?